core/src/main/java/io/opencensus/tags/TagContext.java

```diff
@@ -135,7 +145,7 @@ public Builder clear(TagKey<?> key) {
      * @return a {@code TagContext} with the same tags as this builder.
      */
     public TagContext build() {
-      return new TagContext(new HashMap<TagKey<?>, Object>(tags));
+      return new TagContext(new HashMap<TagKey, Object>(tags));
```

Comment:
But this is just useless. If you simply `return new TagContext(tags);` then the result is the same. I don't understand why do we need this extra copy now?